src/devices/xyzroe.ts

```diff
@@ -0,0 +1,274 @@
+
+import * as exposes from '../lib/exposes';
+import fz from '../converters/fromZigbee';
+import * as legacy from '../lib/legacy';
+import tz from '../converters/toZigbee';
+import {Definition, Tz, Fz, KeyValueAny} from '../lib/types';
+import * as utils from '../lib/utils';
+const e = exposes.presets;
+const ea = exposes.access;
+
+const buttonModesList = {
+    'single click': 0x01,
+    'multi click': 0x02,
+};
+
+const inputLinkList = {
+    no: 0x00,
+    yes: 0x01,
+};
+
+const bindCommandList = {
+    'on/off': 0x00,
+    'toggle': 0x01,
+    'change level up': 0x02,
+    'change level down': 0x03,
+    'change level up with off': 0x04,
+    'change level down with off': 0x05,
+    'recall scene 0': 0x06,
```

Comment:
Please make these all snake case, e.g. `recall_scene_0`

Reply:
ok, but they are used in the interface, I wanted to make it look visually pleasing.

Reply:
![Screenshot 2024-02-18 at 9 48 36 PM](https://github.com/Koenkk/zigbee-herdsman-converters/assets/6440415/c15ff080-0cd0-4a01-82af-897405f417e0)
will snake case look better in the interface? 

just want confirm before commit